### DISCUSSION TITLE
How can I pass RequestIdKey in Identity provider?

### DISCUSSION THREAD

**Discussion by AmitaKhatri** [ThreeMammals/Ocelot#1893] [Q&A]
Hi,

My Ocelot API gateway application is using an identity server for authentication using JWT token. I am using RequestIdKey at the global level to forward the Correlation Id. The correlation id is forwarded in the header to the downstream applications but it is not forwarded while communicating with our identity provider application. Is there any way to achieve that?

**Comment by raman-m** [comment]
Hi Amita!
Welcome to Ocelot world! 🐯 

> but it is not forwarded while communicating with our identity provider application.

Is your identity management app outside of Ocelot domain?
Is/Isn't it a part of Ocelot app?
